get-new-tool-versions/send-slack-notification.ps1

```diff
@@ -35,7 +35,7 @@ param(
 Import-Module $PSScriptRoot/helpers.psm1 -DisableNameChecking
 
 # Create JSON body
-$text = "The following versions of '$toolName' are available to upload: $toolVersion"
+$text = "The following versions of '$toolName' are available, consider adding them to toolset: $toolVersion"
```

Comment:
Will it change the message for other tools as well? I think we need different messages for Xamarin and Go, Python and so on